chromium_src/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h

```diff
@@ -6,12 +6,28 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_BROWSING_DATA_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_BROWSING_DATA_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H_
 
+#include "build/build_config.h"
+
 class BraveBrowsingDataRemoverDelegate;
 
+#if !BUILDFLAG(IS_ANDROID)
+class FalseBoolStub {
+ public:
+  bool operator=(bool) { return false; }
+  explicit operator bool() const { return false; }
+};
+#endif
+
 #define BRAVE_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H \
   friend class BraveBrowsingDataRemoverDelegate;
 
+#define should_clear_sync_account_settings_          \
```

Comment:
please either add a comment that this variable is already under `#if !BUILDFLAG(IS_ANDROID)` or just add an explicit buildflag check for this define-override.

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/25147/commits/750cb5e9330c7faf62cb575d09cc3a14c0fbc5e0